sharding/review/casper_v2.1.md

```diff
@@ -0,0 +1,362 @@
+# Full Casper chain v2.1
+
+## WORK IN PROGRESS!!!!1!!1!
+
+This is the work-in-progress document describing the specification for the Casper+Sharding chain, version 2.1.
+
+In this protocol, there is a central PoS chain which stores and manages the current set of active PoS validators. The only mechanism available to become a validator initially is to send a transaction on the existing PoW main chain containing 32 ETH. When you do so, as soon as the PoS chain processes that block, you will be queued, and eventually inducted as an active validator until you either voluntarily log out or you are forcibly logged out as a penalty for misbehavior.
+
+The primary source of load on the PoS chain is **attestations**. An attestation has a double role:
+
+1. It attests to some parent block in the beacon chain
+2. It attests to a block hash in a shard (a sufficient number of such attestations create a "cross-link", confirming that shard block into the main chain).
+
+Every shard (e.g. there might be 1024 shards in total) is itself a PoS chain, and the shard chains are where the transactions and accounts will be stored. The cross-links serve to "confirm" segments of the shard chains into the main chain, and are also the primary way through which the different shards will be able to talk to each other.
```

Comment:
`there might be 1024 shards in total` -> `there might be 1024 shards initially` to imply that the shards may grow?

Reply:
The number of shards is constant within the scope of the mechanism (ie. adding more would require a hard fork). So IMO better to keep the emphasis of "it's 1024, deal with it".